### PR TITLE
firmware/include update V20.03.00 and fixes

### PR DESCRIPTION
**Pull request by ernunesrh**
Set fwts version to V20.03.00 and update bundled patches.

### CODE REVIEW

File: firmware/include/patches/0001-build-do-not-use-Werror.patch [hunk 1]
```diff
@@ -20,55 +20,55 @@ Signed-off-by: Erico Nunes <ernunes@redhat.com>
...
  	-pthread `pkg-config --cflags glib-2.0 gio-2.0` \
 -	-Wall -Werror -Wextra				\
-+	-Wall  -Wextra				\
- 	-Wno-address-of-packed-member
- 
- bin_PROGRAMS = fwts
++	-Wall -Wextra				\
+ 	-Wno-address-of-packed-member			\
+ 	-Wfloat-equal -Wmissing-declarations		\
+ 	-Wno-long-long -Wredundant-decls -Wshadow	\
```
[the remark this answers is not in the document]
Comment (ernunesrh) [reply firmware/include/patches/0001-build-do-not-use-Werror.patch:45]: Yes, these are just context lines on the patch that appeared as the patch was rebased to apply cleanly on the new version.

[the remark this answers is not in the document]
Comment (emanchado) [reply firmware/include/patches/0001-build-do-not-use-Werror.patch:45]: Ah, right, of course, haha 😅

**Comment by emanchado** [comment]
I don't know why the Travis checks are taking so long/are stuck, but in the meantime, hopefully @cki-bot will be able to test this?

**Comment by emanchado** [comment]
@cki-bot be a good bot and test this

**Comment by emanchado** [comment]
The failures seem unrelated, but I don't know what they mean. @rasibley? It seems that `rt_cert_test` is failing (?), I don't think I had seen it fail before. And the podman test failing on recipe 8104693... ???

I'm ok with the patch, I'd just prefer Rachel to confirm that the test failures have nothing to do with this patch.

**Comment by rasibley** [comment]
@emanchado the rt_test was newly on-boarded it's used to verify the official builds are signed properly, this test is running incorrectly on baselines, since we re-trigger baselines for CI, I will look into fixing this. Also podman was just a timeout, I'll increase the timeout for that test.

**Pull request merged**